docs/web/api-angular/schematics/karma.md

```diff
@@ -6,5 +6,18 @@ Add karma configuration to a workspace
 
 ```bash
 nx generate karma ...
+```
+
+By default, Nx will search for `karma` in the default collection provisioned in `workspace.json`.
+
+You can specify the collection explicitly as follows:
+
+```bash
+nx g @nrwl/angular:karma ...
+```
 
+Show what will be generated without writing to disk:
+
+```bash
+nx g null ... --dry-run
```

Comment:
There is still null here. This should be ng g karma or ng g @nrwl/angular:karma --dry-run